packages/front-end/pages/features/[fid].tsx

```diff
@@ -217,11 +219,58 @@ export default function FeaturePage() {
                 text="Delete feature"
               />
             )}
+            {permissions.createFeatures && (
+              <ConfirmButton
+                onClick={async () => {
+                  await apiCall(`/feature/${data.feature.id}/archive`, {
+                    method: "POST",
+                  });
+                  router.push("/features");
+                }}
+                modalHeader={
+                  isArchived ? "Unarchive Feature" : "Archive Feature"
+                }
+                confirmationText={
+                  isArchived ? (
+                    <>
+                      <p>
+                        Are you sure you want to continue? This will make the
+                        current feature active again.
+                      </p>
+                    </>
+                  ) : (
+                    <>
+                      <p>
+                        Are you sure you want to continue? This will make the
+                        current feature unactive.
+                      </p>
+                    </>
+                  )
+                }
+                cta={isArchived ? "Unarchive" : "Archive"}
+                ctaColor="danger"
+              >
+                <button className="dropdown-item">
+                  {isArchived ? "Unarchive" : "Archive"} Feature
+                </button>
+              </ConfirmButton>
+            )}
           </MoreMenu>
         </div>
       </div>
 
-      <h1>{fid}</h1>
+      <div className="row align-items-center mb-3">
+        <h1 className="col-auto mb-0">{fid}</h1>
+        {isArchived && (
+          <div
+            className="badge badge-secondary mx-3 h4 mb-0"
```

Comment:
I tried that, but then the Archived status no longer appears in the same row as the feature id (name). I basically just used the same styling as the status indicators do in experiments. The only reason I didn't make a different component for this in features is because we only have one unique type of feature, archived feature, as compared to more types and hence more banners for experiments

Reply:
Looking at the page, there's a lot of badges happening in the same area:
![image](https://user-images.githubusercontent.com/1087514/173193873-63e82531-1462-4271-955b-275bf05d1722.png)

I'm wondering if it will be cleaner to move this next to the Version dropdown instead:
![image](https://user-images.githubusercontent.com/1087514/173193917-4d1330ce-f8c3-49ea-b498-2018693992d4.png)

Reply:
As an alternative to the badge, this is what we do at the top of archived metric pages.  I might like this approach better since it's less likely someone will miss it.  I can see a user getting frustrated trying to figure out why their feature isn't being returned from the API and not noticing the tiny "archived" badge on the page.  

```tsx
      {metric.status === "archived" && (
        <div className="alert alert-secondary mb-2">
          <strong>This metric is archived.</strong> Existing references will
          continue working, but you will be unable to add this metric to new
          experiments.
        </div>
      )}
```

For features, we could use the language:
> **This feature is archived.** It will not be included in API responses or Webhook payloads.